charts/fluent-bit/values.yaml

```diff
@@ -78,6 +78,9 @@ securityContext: {}
 service:
   type: ClusterIP
   port: 2020
+# loadBalancerClass: service.k8s.aws/nlb
+# loadBalancerSourceRanges:
+#   - 10.0.0.0/8
```

Comment:
Could you leave the values defined but empty as it's far easier to find template issues that way. You can add examples in as separate comments if you think they're needed.

Reply:
Absolutely. Update incoming.